week1/assignment/가계부 💸/style.css

```
@@ -0,0 +1,204 @@
+@font-face {
+    font-family: 'Pretendard';
+    src: url('./font/Pretendard-Medium.otf') format('opentype');
+}
+
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: Pretendard, sans-serif;
+    box-sizing: border-box;
+}
+
+html {
+    background-color: #d9d9d9;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    width: 100vw;
+    min-height: 100vh;
+    margin: auto;
+    padding: 10px;
+    background-color: #f8f8f8;
+}
+
+button {
+    cursor: pointer;
+}
+
+header {
+    padding: 20px;
+    background-color: #ffffff;
+    border-radius: 10px;
+}
+
+h1 {
+    font-size: 20px;
+    font-weight: 500;
+}
+
+#asset {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 16px;
+    align-items: center;
+    padding: 20px;
+    background-color: #ffffff;
+    border-radius: 12px;
+}
+
+.subtitle {
+    font-size: 20px;
+    font-weight: 500;
+}
+
+#money {
+    font-size: 20px;
+}
+
+#today {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    width: 160px;
+    align-items: center;
+}
+
+#today img {
+    width: 16px;
+    height: 16px;
+}
+
+.income {
+    color: #4d7cfe;
+}
+
+.spending {
+    color: #fe4d4d;
+}
+
+#main {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    max-height: calc(100vh - 300px);
+    padding: 20px;
+    background-color: #ffffff;
+    border-radius: 12px;
+}
+
+#date {
+    display: flex;
+    justify-content: space-between;
+    margin: 20px 0 10px 0;
+    width: 160px;
+}
+
+#date button {
+    border: none;
+    background: transparent;
+}
+
+#money-type {
+    display: flex;
+    justify-content: end;
+    font-size: 14px;
+}
+
+#money-type input {
+    display: none;
+}
+
+#money-type label {
+    margin-left: 10px;
+    padding: 5px 20px 5px 20px;
+    color: #777777;
+    background-color: #f2f2f2;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+#money-type input[type=checkbox]:checked+label {
+    margin-left: 10px;
+    padding: 5px 20px 5px 20px;
```

```suggestion
padding: 5px 20px;
```
이렇게 한 번에 가능하겠네요 !! ㅎㅎ